include/linux/soundwire/sdw.h

```diff
@@ -981,6 +981,7 @@ struct sdw_stream_runtime {
  * @params: Current bus parameters
  * @stream_refcount: number of streams currently using this bus
  * @btp_stream_refcount: number of BTP streams currently using this bus (should
+ * @bpt_stream: pointer stored for convenience.
  * be zero or one, multiple streams per link is not supported).
```

Comment:
You cut the description for the previous member in half ;)
And it is not for convenience, the bpt_stream pointer is needed to be stored to handle BPT/BRA

Reply:
fixed